apollo-router/src/apollo_studio_interop/test.rs

```diff
@@ -0,0 +1,1419 @@
+#[cfg(test)]
+mod tests {
```

Comment:
Delete these two lines and the matching module close at the end. They are already in module `tests` by virtue of the file name.